docs/_plugins/p2export.md

```diff
@@ -0,0 +1,216 @@
+
+* `feature` – Specifies a feature
+*  p2 namespace – P2 namespaces are used to specify requirements for the feature. For example, `osgi.bundle` specifies a bundle requirement and `java.package` specifies a package requirement. This way, any P2 eclipse namespace can be added.
+
+These pseudo namespaces use the attributes `name` and `version` to specify the name and version _range_ of the requirement. They do not require a filter since p2 lacks this concept.
+
+The following example shows how to specify a feature requirement:
+
+    Require-Capability: \
+        feature; \
+            name= com.example.my.first.feature; \
+            version='[1.1.0,1.1.0]'
+
+If no version is specified in a feature, it will default to the version of the requiring feature. This means that if you specify a feature requirement without a version, the version of the requiring feature will be used as the version requirement for the required feature.
+
+## Categories
+
+Features can be grouped in categories.A feature can be a member in a category by specifying the `Bundle-Category` header. The value of this header is the identity of the category. For example, the following header specifies that the feature belongs to the category "bndtools" and "java":
+
+    Bundle-Category: bndtools, java
+
+The release bndrun file provide the details of the category. The following example shows how to specify the details of the categories can be specified with the `-categories` instruction in the project's bnd.bnd file:
+
+    -categories \
+        bndtools; \
+            label=Bndtools; \
+            description=Bndtools is a set of tools for OSGi development,\
+        java; \
+            label=Java ©; \
+            description=The Java language
+
+## Output
+
+The output will be a self contained p2 repository contained in a JAR file. it contains the following:
+
+* `content.jar` – This is the metadata of the repository
+* `artifacts.jar` – A list of all the artifacts in the repository and rules how to map an id to a file in the p2 archive.
+* `plugins/` – A directory with all the bundles in the repository
+* `features/` – A directory with all the features in the repository
+
+The repository can be used as a drop-in replacement for the Eclipse p2 repository. It can be used to update an Eclipse installation or to install a new Eclipse installation.
+
+## Sample Setup
+
+In the workspace's build.bnd file:
+
+    -plugin \
+        aQute.p2.export.P2Exporter
+
+    Bundle-Version: 1.0.0
+    Bundle-Vendor: Example
+    Bundle-License: Apache-2.0
+    Bundle-Copyright: Public domain
+
+
+In project X's bnd.bnd file:
+
+    -export \
+        release.bndrun; \
+            name=myrepo.jar; \
+            features="a.bndrun,b.bndrun"
+
+    -categories \
+        java; \
+            label=Java ©; \
+            description=The Java language
+
+In the a.bndrun file:
```

Comment:
Do you also generate feature.xml out of these information?

Reply:
Yes.